### DISCUSSION TITLE
You cannot 'userLoaderData' in an error boundary

### DISCUSSION THREAD

**Discussion by mirandaa1** [remix-run/remix#3734] [General]
I get this error while I try to regitser a user..
<img width="664" alt="image" src="https://user-images.githubusercontent.com/36954358/178961523-aa7d7e2b-bcc3-4e5b-9175-d9534f12ec77.png">

**Comment by kiliman** [comment]
Yes, because the error could have happened in the loader (or prior to your loaders being called), you do not have access to that data. `<ErrorBoundary/>` is for *unexpected* errors, so your app may not be in a usable state.

If your app knows how to handle this error, then you should wrap the code in `try/catch`. You can either handle the error, or *throw* a new Response which will end up in your `<CatchBoundary/>` where you can `useCatch` to access the body of your Response.
